contents/docs/integrate/ingest-live-data.mdx

```diff
@@ -121,4 +124,29 @@ import ReactNativeIdentify from './client/react-native/snippets/identify.mdx'
   <TabPane tab="React Native" key="5">
     <ReactNativeIdentify />
   </TabPane>
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+# Event ingestion nuances
+
+It's top-priority for us that events are fully processed and saved as soon as possible. However, there is a class of events which we _deliberately_ process with a slight delay. Specifically, an event is delayed by around a minute if it fits those three conditions:
```

Comment:
I wouldn't currently state this openly - it's not a commitment we're ready to make. We're not a real time data platform, even though we strive to be as fast as possible, and I'd rather not have users hold us to this yet. It's a bit of a contentious point internally - if should we commit to this or not.

Reply:
Why _aren't_ we real time? I can reword from "top-priority" to "a priority", but I think if we find the extra delay worth explaining, it means there _are_ some expectations we're trying to fulfill.